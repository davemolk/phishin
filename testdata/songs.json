{"success":true,"total_entries":942,"total_pages":48,"page":1,"data":[{"id":84,"slug":"billy-breathes","title":"Billy Breathes","alias":null,"original":true,"artist":null,"lyrics":"[Verse]\nTumbling greens\nA pickup screams\nAlone above the square\nOh sing softly\nAbove the trees\nWhere Billy breathes\nWe float upon the air\nSoftly sing sweet songs\nSilent scenes\nIn motion means\nI’ll wake you when we’re there\nOh sing softly\nTime, it seems\nIn broken dreams\nTo sleep beside the stair","tracks_count":64,"updated_at":"2021-05-09T05:13:22Z"},{"id":173,"slug":"costume-contest","title":"Costume Contest","alias":null,"original":true,"artist":null,"lyrics":"[Banter varies by performance]","tracks_count":3,"updated_at":"2021-05-09T01:09:27Z"}]}