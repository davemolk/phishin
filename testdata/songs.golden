Title:          Original Artist:  TracksCount:
Billy Breathes  Phish             64
Arc             Arctic Monkeys    0

Total Entries: 942  Total Pages: 48  Result Page: 1
